packages/components/src/templates/badge.stories.ts

```diff
@@ -32,15 +32,21 @@ export const ButtonWithBadge = {
         </sd-button>
         <sd-button variant="secondary" size="md">
           <sd-icon name="system/phone"></sd-icon>
-          <sd-badge variant="error" size="md">4</sd-badge>
+          <sd-badge variant="error" size="md">
+            <span class="sr-only">Calls</span>
+            4
+          </sd-badge>
         </sd-button>
         <sd-button variant="tertiary" size="md">
           <sd-icon name="system/bell"></sd-icon>
-          <sd-badge class="mt-2.5 mr-2.5" variant="success" size="md">5</sd-badge>
+          <sd-badge class="mt-2.5 mr-2.5" variant="success" size="md">
+            <span class="sr-only">Notifications</span>
+            5
```

Comment:
I would call it out "5 Notifications" instead of "Notifications 5"?

Reply:
Good point. Updated